LCAPI.Core/LCAPI.Core.csproj

```diff
@@ -1,22 +1,53 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>netstandard2.1</TargetFramework>
+        <TargetFramework>net48</TargetFramework>
         <AssemblyName>LCAPI.Core</AssemblyName>
+        <RootNamespace>LCAPI.Core</RootNamespace>
         <Description>A core modding library for Lethal Company.</Description>
         <Version>1.0.0</Version>
+        <DebugType>embedded</DebugType>
         <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
         <LangVersion>latest</LangVersion>
+        <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\LCAPI.Core.xml</DocumentationFile>
+        <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)LCAPI.ruleset</CodeAnalysisRuleSet>
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="BepInEx.Analyzers" Version="1.*" PrivateAssets="all"/>
-        <PackageReference Include="BepInEx.Core" Version="5.*" />
-        <PackageReference Include="BepInEx.PluginInfoProps" Version="1.*"/>
-        <PackageReference Include="UnityEngine.Modules" Version="2022.3.9" IncludeAssets="compile" />
+        <PackageReference Include="BepInEx.Analyzers" Version="1.*" PrivateAssets="all" ExcludeAssets="RUNTIME" />
+        <PackageReference Include="BepInEx.Core" Version="5.*" ExcludeAssets="RUNTIME" />
+        <PackageReference Include="BepInEx.PluginInfoProps" Version="1.*" ExcludeAssets="RUNTIME" />
+        <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.507" PrivateAssets="all" IncludeAssets="runtime; build; native; contentfiles; analyzers; buildtransitive" ExcludeAssets="RUNTIME" />
+        <PackageReference Include="UnityEngine.Modules" Version="2022.3.9" IncludeAssets="compile" ExcludeAssets="RUNTIME" />
     </ItemGroup>
 
     <ItemGroup Condition="'$(TargetFramework.TrimEnd(`0123456789`))' == 'net'">
-        <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.2" PrivateAssets="all"/>
+        <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.2" PrivateAssets="all" ExcludeAssets="RUNTIME" />
     </ItemGroup>
+    
+    <ItemGroup>
+        <Reference Include="Assembly-CSharp" HintPath="$(LC_REFERENCES)\Assembly-CSharp-Publicized.dll" Private="False" />
```

Comment:
Let's keep this to the default Assembly-CSharp for now. Publicised dll can be sorted out once the rest of the team is onboard with it.